frontend/src/pages/user/shop/Shop.tsx

```diff
@@ -1,19 +1,39 @@
 import { Col, Row } from 'react-bootstrap';
+import { useQuery } from '@tanstack/react-query';
 
 import GoodsItem from '@components/GoodsItem';
-
-import goodsData from '@pages/discover/goodsData.json';
+import { Props } from '@components/GoodsItem';
+import { CheckStatus } from '@lib/CheckStatus';
 
 const Shop = () => {
+  const { status, data } = useQuery({
```

Comment:
u can rename it if the word data is too precious. ( cus i saw ``d`` at the end of this file)
```tsx
const { status, data : shopData } = useQuery({
```